ProcessMaker/Notifications/ErrorExecutionNotification.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace ProcessMaker\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\BroadcastMessage;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+use ProcessMaker\Nayra\Contracts\Bpmn\TokenInterface;
+
+class ErrorExecutionNotification extends Notification
+{
+    use Queueable;
+
+    private $tokenInterface;
+    private $message;
+    private $errorHandling;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct(TokenInterface $tokenInterface, $message = '', $errorHandling = [])
+    {
+        $this->tokenInterface = $tokenInterface;
+        $this->message = $message;
+        $this->errorHandling = $errorHandling;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        $via = [];
+        if ($this->errorHandling['inapp_notification'] === true) {
+            $via[] = NotificationChannel::class;
+        }
+        if ($this->errorHandling['email_notification'] === true) {
+            $via[] = 'mail';
+        }
+        return $via;
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        $data = $this->toArray($notifiable);
+        return (new MailMessage)
+            ->line($data['message'])
+            ->action('Notification Action', url($data['url']))
+            ->line($this->message);
```

Comment:
Lets customize the email
- The Notification Action should be "View error in app"
- Try adding `->error()`. See https://laravel.com/docs/8.x/notifications#error-messages
- add `->greeting("Request #ID Execution Error")` and `->subject("Request #ID Execution Error")` and replace #ID with the request id